draft-ietf-privacypass-protocol.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Privacy Pass Protocol Specification"
 abbrev: PP protocol
-docname: draft-ietf-privacypass-protocol-latest
```

Comment:
This typically stays `latest` -- are you trying to build `-01`? If so, let's chat offline and I can help you figure it out!